scripts/exgfs_wave_init.sh

```diff
@@ -203,6 +203,17 @@ source "${HOMEgfs}/ush/preamble.sh"
     fi
   done
 
+# Copy to other members if needed
```

Comment:
Does this need to be linked this many times or can we simply just grab the single mod_def file in the forecast job?

Reply:
When the forecast runs (and other jobs?), they don't really know how to find the deterministic member. Plus the GFS and GEFS (currently) label the deterministic differently (GEFS uses `mem000`, whereas GFS is in an entirely different `RUN` from EnKF).

Reply:
Thanks for the explanation